tests/sndrcv_tls_gtls_servercert_gtls_clientanon_legacy.sh

```diff
@@ -36,18 +35,24 @@ export RSYSLOG_DEBUGLOG="log2"
 generate_conf 2
 #export TCPFLOOD_PORT="$(get_free_port)"
 add_conf '
+global( defaultNetstreamDriverCAFile="'$srcdir/tls-certs/ca.pem'"
+)
+
 # Note: no TLS for the listener, this is for tcpflood!
-$ModLoad ../plugins/imtcp/.libs/imtcp
+module(	load="../plugins/imtcp/.libs/imtcp")
```

Comment:
this needs to be done via legacy config statements, as before. The test checks if the legacy statements work ;-)  (thus the `-legacy` inside the test name).